versions/1/manifest.spec.yml

```diff
@@ -106,6 +106,16 @@ spec:
         - "1.0.0-SNAPSHOT"
         - "1.0.0-dev.1"
   properties:
+    elasticsearch:
+      description: Elasticsearch cluster requirements
```

Comment:
L110 and L115 have the same comment, but I guess that they have/should have different meaning?